ui/app/routes/vault/cluster/dashboard.js

```diff
@@ -9,10 +9,32 @@ import { hash } from 'rsvp';
 
 export default class VaultClusterDashboardRoute extends Route {
   @service store;
+  @service version;
+
+  async getVaultConfiguration() {
+    try {
+      const adapter = this.store.adapterFor('application');
+      const configState = await adapter.ajax('/v1/sys/config/state/sanitized', 'GET');
+      return configState.data;
+    } catch (e) {
+      return e.httpStatus;
+    }
+  }
 
   model() {
     return hash({
       secretsEngines: this.store.query('secret-engine', {}),
+      version: this.version,
+      vaultConfiguration: this.getVaultConfiguration(),
     });
   }
+
+  setupController(controller, resolvedModel) {
```

Comment:
I'm curious why you opted to put the `versionHeader` onto the controller instead of making it a property on the model?

Reply:
Another approach could be a getter to format `versionHeader` and then adding that as a property to the model instead of the version. (I didn't see anywhere that you used `this.model.version`)

Reply:
Good point! I will put it directly in the model since the version isn't being used anywhere else.